lib/externalGraderLocal.js

```diff
@@ -94,7 +96,12 @@ class Grader {
                     stderr: true,
                 }, (err, stream) => {
                     if (ERR(err, callback)) return;
-                    container.modem.demuxStream(stream, process.stdout, process.stderr);
+                    const out = byline(stream);
```

Comment:
`byline` apparently skips empty lines by default. I'm not sure if we want them? https://github.com/jahewson/node-byline#empty-lines

Reply:
Good catch, someone is bound to complain if they're missing empty lines. I'll add that.

Reply:
On second though, I'm maintaining this behavior; keeping empty lines was adding an empty line to the end of the logs which I thought looked bad.

Reply:
Sounds good